OpenEdXMobile/src/main/java/org/edx/mobile/module/analytics/FirebaseAnalytics.java

```diff
@@ -393,4 +393,56 @@ public void identifyUser(String userID, String email, String username) {
     public void resetIdentifyUser() {
         tracker.setUserId(null);
     }
+
+    @Override
+    public void trackAppRatingDialogViewed(String versionName) {
+        final FirebaseEvent event = new FirebaseEvent(Events.APP_REVIEWS_VIEW_RATING,
+                Values.APP_REVIEWS_VIEW_RATING);
+        event.putString(Keys.APP_VERSION, versionName);
+        logFirebaseEvent(event.getName(), event.getBundle());
+    }
+
+    @Override
+    public void trackAppRatingDialogCancelled(String versionName) {
+        final FirebaseEvent event = new FirebaseEvent(Events.APP_REVIEWS_DISMISS_RATING,
+                Values.APP_REVIEWS_DISMISS_RATING);
+        event.putString(Keys.APP_VERSION, versionName);
+        logFirebaseEvent(event.getName(), event.getBundle());
+    }
+
+    @Override
+    public void trackUserSubmitRating(String versionName, int rating) {
+        final FirebaseEvent event = new FirebaseEvent(Events.APP_REVIEWS_SUBMIT_RATING,
+                Values.APP_REVIEWS_SUBMIT_RATING);
+        event.putString(Keys.APP_VERSION, versionName);
+        event.putLong(Keys.RATING, rating);
```

Comment:
You can make and use a `putInt` function in FirebaseEvent class instead of using the `putLong`.